client/src/routes/index.js

```diff
@@ -43,6 +43,7 @@ export default () => {
       autoRefreshToken={true}
       initConfig={{
         pkceMethod: 'S256',
+        checkLoginIframe: false,
```

Comment:
That was the cause of the refreshing bug yesterday.

Reply:
Nice find 👍